message-collector/src/main/java/no/vegvesen/ixn/federation/messagecollector/MessageCollector.java

```diff
@@ -1,7 +1,9 @@
 package no.vegvesen.ixn.federation.messagecollector;
 
 import no.vegvesen.ixn.federation.model.GracefulBackoffProperties;
+import no.vegvesen.ixn.federation.model.ListenerEndpoint;
 import no.vegvesen.ixn.federation.model.Neighbour;
```

Comment:
Remove unused imports. Neighbour is no longer here :-)

Reply:
Done :)